Automattic/jetpack
# Changelogger: Allow comments in non-interactive mode

## Proposed changes:


When using Jetpack Changelogger in non-interactive mode, I can create a changelog entry without issue. When I provide a comment (using significance `patch`) instead of an entry, the following error occurs:

```
Entry must be specified in non-interactive mode.
```
Here is the raw command using Jetpack Changelogger v3.3.0

```
composer exec -- changelogger "add" "-f" update-add_gitignore_to_package" "-s" "patch" "-t" "added" "-c" "Minimal change that was missed during bootstrap, no changelog necessary." "-n"
```

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Navigate to any project, for example:
```
cd projects/packages/changelogger
```
2. Execute a non interactive changefile addition with a comment
```
composer exec -- changelogger add -f file-name -s patch -t added -c "this is a comment" -n
```
3. The command should succeed. Note it fails when on `trunk`
